docs/open-telemetry.asciidoc

```diff
@@ -15,28 +16,67 @@
 :ot-collector:  https://opentelemetry.io/docs/collector/getting-started/
 :ot-dockerhub:  https://hub.docker.com/r/otel/opentelemetry-collector-contrib
 
-https://opentelemetry.io/docs/concepts/what-is-opentelemetry/[OpenTelemetry] is a set
-of APIs, SDKs, tooling, and integrations that enable the capture and management of
-telemetry data from your services for greater observability. For more information about the
+{ot-what}[OpenTelemetry] is a set of APIs, SDKs, tooling, and integrations that enable the capture and management of
+telemetry data from your services and applications. For more information about the
 OpenTelemetry project, see the {ot-spec}[spec].
 
-Elastic OpenTelemetry integrations allow you to reuse your existing OpenTelemetry
-instrumentation to quickly analyze distributed traces and metrics to help you monitor
-business KPIs and technical components with the {stack}.
-
 [float]
-[[open-telemetry-native]]
-==== APM Server native support of OpenTelemetry protocol
+== OpenTelemetry and the {stack}
+
+image::./images/apm-otel-architecture.png[Architecture of Elastic APM with OpenTelemetry]
+
+Elastic integrates with OpenTelemetry, allowing you to reuse your existing instrumentation
+to easily send observability data to the {stack}.
+There are four ways to integrate OpenTelemetry with the {stack}:
+
+**Elastic OpenTelemetry Bridge**
+
+To unlock the full power of the {stack}, use the Elastic APM OpenTelemetry Bridge,
+currently supported by the Java, Python, and Node.js agents.
```

Comment:
@estolfo same question about .NET here. Also, I went with, `To unlock the full power of the {stack}`, but it'd be great to include a _real_ reason why someone would want to use the bridge over direct.